src/main/java/controller/LadderGame.java

```diff
@@ -25,8 +27,13 @@ public void play() {
 
     private void run() {
         participants = recruitParticipants();
+        result = decideResult(participants.getParticipantsCount());
```

Comment:
```suggestion
        result = decideResult(participants.count());
```
count나 size로 작성해도 충분히 의미가 표현될 것 같아요~

Reply:
그러네요......!!!!!